spec/unit/facter/network_nexthop_ip_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe Facter.fact(:network_nexthop_ip) do
+  before { Facter.clear }
+  after { Facter.clear }
+
+  describe 'network_nexthop_ip' do
+    context' returns gateway' do
+      before do
+        allow(Facter.fact(:kernel)).to receive(:value).and_return(:linux)
+        allow(Facter::Util::Resolution).to receive(:which).with('ip').and_return('/usr/bin/ip')
+      end
+      let(:facts) { { kernel: :linux } }
```

Comment:
I don't think this is really a thing in fact testing.

Reply:
I copied it from https://github.com/voxpupuli/puppet-systemd/blob/master/spec/unit/facter/systemd_version_spec.rb#L13 but wasn't really sure about it.